### PR TITLE
Add Github Action to push images from boba-develop

### PR DESCRIPTION
**Pull request by boyuan-chen**
- A workflow builds images from the boba-develop branch and pushes to docker hub.
- A workflow builds images from the release and pushes to docker hub.

### CODE REVIEW

File: ops-bedrock/docker-compose.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ volumes:
 
 services:
   l1:
```
Comment (jyellick) [inline ops-bedrock/docker-compose.yml]: I'm not so sure we want these images specified in the compose? The compose file is used as part of the CI tests, so I think we need it to be referencing the local images, or, we need to modify the way they're tagged?

File: boba-chain-ops/ether/migrate_test.go [hunk 2]
```diff
@@ -356,21 +356,6 @@ func TestMigrateBalancesRandomMissing(t *testing.T) {
 		err := doMigration(g, addresses, allowances, big.NewInt(0), false)
 		require.ErrorContains(t, err, "unknown storage slot")
 	}
-
-	for i := 0; i < 100; i++ {
```
Comment (jyellick) [inline boba-chain-ops/ether/migrate_test.go:360]: Why is this removed?

**Review by jyellick — COMMENTED** [review]
Publishing bits look good

File: boba-chain-ops/ether/migrate_test.go (hunk 2 above)
Comment (boyuan-chen) [reply boba-chain-ops/ether/migrate_test.go:360]: It's duplicate with the above code.

File: ops-bedrock/docker-compose.yml (hunk 1 above)
Comment (boyuan-chen) [reply ops-bedrock/docker-compose.yml]: ok, so I remove the tag in docker-compose and use docker with Dockerfile to build images with the right tag.

**Review by jyellick — APPROVED** [review]

**Review by mmontour1306 — APPROVED** [review]
